stable/filebeat/templates/daemonset.yaml

```diff
@@ -117,12 +121,25 @@ spec:
         hostPath:
           path: /var/lib/docker/containers
       - name: filebeat-config
-        secret:
-          secretName: {{ template "filebeat.fullname" . }}
+        configMap:
+        {{- if .Values.existingConfigmap }}
```

Comment:
What's the benefit of adding a existing config map instead of the user just overriding this configuration?

Reply:
I can not tell of the particular benefit, though most of the charts allow this (seems to be a good practice)... It doesn't harm.

Reply:
additional flexibility never harms in case it's cheap to implement

Reply:
YAGNI, don't change things unless you have a good reason to.